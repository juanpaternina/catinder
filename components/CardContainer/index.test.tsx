import { Text } from "react-native";
import { render, screen } from "@testing-library/react-native";
import CardsContainer from "./index";

describe("CardsContainer", () => {
  it("renders correctly", () => {
    render(
      <CardsContainer>
        <Text>Test</Text>
      </CardsContainer>
    );
    expect(screen.getByText("Test")).toBeTruthy();
    expect(screen).toMatchSnapshot();
  });
});
