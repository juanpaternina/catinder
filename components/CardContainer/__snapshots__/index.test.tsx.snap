// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardsContainer renders correctly 1`] = `
<View
  style={
    {
      "height": 440,
      "marginBottom": 58,
      "width": "100%",
    }
  }
>
  <Text>
    Test
  </Text>
</View>
`;
